manual.docbook

```diff
@@ -2386,8 +2386,8 @@
           loaded into your Sound Library.  Then, select
 
           <menuchoice>
-            <guimenuitem>Instruments</guimenuitem>
-            <guimenuitem>"Export library"</guimenuitem>
+            <guimenu>Drumkits</guimenuitem>
```

Comment:
You accidentally changed the opening tag as well. Could you make it a `guimenuitem` again in order for the document to compile.